rows_test.go

```diff
@@ -54,7 +54,7 @@ func TestTextRows(t *testing.T) {
 }
 
 func TestTextRowsQuoted(t *testing.T) {
-	buf := bytes.NewReader([]byte("text\nArray(String)\n['Quote: \"here\"']"))
+	buf := bytes.NewReader([]byte("text\nArray(String)\n['Quote: \"here\"']\n"))
```

Comment:
Does clickhouse return the data with a trailing \n? if not we should leave the tests as is

Reply:
yes, clickhouse output with the TabSeparatedWithNamesAndTypes format ends with `\n`